controllers/errors.js

```diff
@@ -1,8 +1,25 @@
-exports.handle404s = (req, res, next) => {
-  res.status(404).send({ msg: "Cannot find what you wanted" });
+exports.handleCustomErrors = (req, res, next) => {
+  res
+    .status(404)
+    .send({ msg: "You tried to look for something and it wasn't found" });
+};
+
+exports.handle400s = (err, req, res, next) => {
+  if (err.status === 400) {
+    res.status(400).send({ msg: "Bad request" });
+  } else {
+    next(err);
+  }
 };
 
 exports.handle500s = (err, req, res, next) => {
-  console.log(err);
-  res.status(500).send({ msg: "Error in code somewhere" });
+  if (err.status === 500) {
+    res.status(500).send({ msg: "Error in code somewhere" });
+  } else {
+    next(err);
+  }
+};
+
+exports.handle404s = (err, req, res, next) => {
+  res.status(404).send({ msg: "Cannot find what you wanted" });
```

Comment:
Here you are overwriting the actual msg and status code with your own. You should be sending back the err.status and err.msg. Your test is only passing because of this. You will need to edit your models.articles to meet the test. 

Additionally I would consider using clearer messages altogether. Currently your 400 & 404 messages read almost identically, consider what each error is actually telling you and be descriptive Ie 404 - Path not found :)